src/java.base/share/conf/security/java.security

```diff
@@ -1309,3 +1309,11 @@ jdk.io.permissionsUseCanonicalPath=false
 # System value prevails. The default value of the property is "false".
 #
 #jdk.security.allowNonCaAnchor=true
+
+#
+# The default Character set name (java.nio.charset.Charset.forName()) for
+# converting TLS ALPN values between byte arrays and Strings. Older versions
+# of JDK used UTF-8.
```

Comment:
It would be useful to mention until which version it was using UTF-8 and from which one ISO_8859_1.
In this PR it is clear that the switch is for JDK 16.
But if some developer reads java.security file later (let's say JDK 17) it would be hard to find this informartion.

Reply:
Hmmm...not sure how to resolve this since we don't have commitments from the Sustaining/OpenJDK folks on if/when this will be included in their releases.

Reply:
Consulted with the Sustaining Team, they requested that this note not be included, as the versions are not yet known.